docs/en/README.md

```diff
@@ -0,0 +1,57 @@
+
+## Highlighted Features
+
+1. **Fully POSIX-compatible**: Use like a local file system, seamlessly docking with existing applications, no business intrusion.
+2. **Fully Hadoop-compatible**: JuiceFS [Hadoop Java SDK](hadoop_java_sdk.md) is compatible with Hadoop 2.x and Hadoop 3.x. As well as variety of components in Hadoop ecosystem.
+3. **S3-compatible**:  JuiceFS [S3 Gateway](s3_gateway.md) provides S3-compatible interface.
+4. **Cloud Native**: JuiceFS provides [Kubernetes CSI driver](how_to_use_on_kubernetes.md) to help people who want to use JuiceFS in Kubernetes.
+5. **Sharing**: JuiceFS is a shared file storage that can be read and written by thousands clients.
+6. **Strong Consistency**: The confirmed modification will be immediately visible on all servers mounted with the same file system .
+7. **Outstanding Performance**: The latency can be as low as a few milliseconds and the throughput can be expanded to nearly unlimited. 
+8. **Data Encryption**: Supports data encryption in transit and at rest, read [the guide](encrypt.md) for more information.
+9. **Global File Locks**: JuiceFS supports both BSD locks (flock) and POSIX record locks (fcntl).
+10. **Data Compression**: JuiceFS supports use [LZ4](https://lz4.github.io/lz4) or [Zstandard](https://facebook.github.io/zstd) to compress all your data.
+
+## Table of content
+
+- **Introduction**
+  - [What is JuiceFS](introduction.md)
+  - [JuiceFS Technical Architecture](architecture.md)
+  - [How JuiceFS store files](how_juicefs_store_files.md)
+  - [How to Setup Object Storage](how_to_setup_object_storage.md)
+  - [Metadata Engines for JuiceFS](databases_for_metadata.md)
+- [Quick Start Guide](quick_start_guide.md)
+- **Basic Usage**
+  - [Use JuiceFS on Linux](juicefs_on_linux.md)
+  - [Use JuiceFS on macOS](juicefs_on_windows)
+  - [Use JuiceFS on Windows](juicefs_on_macos.md)
+  - [Use JuiceFS on Docker](juicefs_on_docker.md)
+  - [Use JuiceFS on Kubernets](juicefs_on_kubernets.md) 
+  - [Use JuiceFS on Hadoop](hadoop_java_sdk.md)
+  - [JuiceFS enable S3 Gateway](s3_gateway.md)
+  - [JuiceFS client compilation and upgrade](client_compile_and_upgrade.md)
+- [Command Reference](command_reference.md)
+- **Advanced Topic**
+  - [Redis best practices](redis_best_practices.md)
+  - [JuiceFS benchmark](benchmark.md)
+  - [POSIX Compatibility](posix_compatibility.md)
+  - [JuiceFS cache management](cache_management.md)
+  - [JuiceFS operations profiling](operations_profiling.md)
+  - [JuiceFS fault diagnosis and analysis](fault_diagnosis_and_analysis.md)
+  - [FUSE mount options](fuse_mount_options.md)
+  - [JuiceFS sync accounts between multiple hosts](sync_accounts_between_multiple_hosts.md)
+  - [Comparison with others](comparison_with_others.md)
+  - [Uasge tracking](usage_tracking.md)
+- [Scenarios & Cases](case/README.md)
+- [FAQ](FAQ.md)
```

Comment:
Link is broken

Reply:
The directory `case` will be remove and a new file case.md for Scenarios & Cases articles index will be created.